Lib/opcode.py

```diff
@@ -336,6 +336,17 @@ def jabs_op(name, op, entries=0):
 _specialized_instructions = [
     opcode for family in _specializations.values() for opcode in family
 ]
+
+_empty_slot = [slot for slot, name in enumerate(opname) if name.startswith("<")]
+for spec_op, specialized in zip(_empty_slot, _specialized_instructions):
```

Comment:
I don't think that we want the specialized forms in `opmap` and `opname`, as they are internal and likely to change between micro versions.

Could you move this code to the dis module?

Reply:
Finish :)